wallet-new/src/Cardano/Wallet/API/V1/Transactions.hs

```diff
@@ -29,4 +29,10 @@ type API = Tags '["Transactions"] :>
                         :> Summary "Estimate the fees which would originate from the payment."
                         :> ReqBody '[ValidJSON] Payment
                         :> Post '[ValidJSON] (WalletResponse EstimatedFees)
+    :<|> "transactions" :> "certificates"
+                        :> Summary "Redeem a certificate"
+                        :> CaptureWalletId
+                        :> CaptureAccountId
+                        :> ReqBody '[ValidJSON] Redemption
+                        :> Post '[ValidJSON] (WalletResponse Transaction)
```

Comment:
This API structure is called like:

```
transactions/certificates/3/321234
```

This is a little odd an difficult to read. The previous URL structure looked like:

```
wallets/3/accounts/321234/certificates
```

which is clearer on what those numbered parameters actually mean.

I think the route structure should include the `wallets/:walletId/accounts/:accountIndex/` "prefixes" or they should be included as part of the `Redemption` datatype. Folding them into the `Redemption` datatype would make for prettier routes.

Reply:
LOL, Looks like y'all came to the same conclusion!

Reply:
Then I guess it's probably our most acceptable solution :D